Utils/Functions/Apps/Resources/script.js

```diff
@@ -1,36 +1,41 @@
 /**
-* AppResource prototype
-* @constructor
-*/
-function AppResource() {
-    this._appsManager = Bean("repositoryManager");
-}
+ * AppResource class
+ */
+// eslint-disable-next-line no-unused-vars
+class AppResource {
+    constructor() {
+        this.appsManager = Bean("repositoryManager");
+    }
 
-/**
-* sets application
-* @param {string} application application of the resource
-* @returns {AppResource} AppResource object
-*/
-AppResource.prototype.application = function (application) {
-    this._application = application;
-    return this;
-}
+    /**
+     * Sets the application containing the resources
+     *
+     * @param {string} application The application with the resource
+     * @returns {AppResource} The AppResource object
+     */
+    application(application) {
+        this._application = this.appsManager.getApplication(application);
 
-/**
-* returns resource
-* @param {string} resourceName name of the resource
-* @returns {Resource} found resource
-*/
-AppResource.prototype.get = function (resourceName) {
-    var application = this._appsManager.getApplication(this._application);
-    var foundResource = null;
-    if (application != null && application.resources != null) {
-        application.resources.forEach(function (resource) {
-            if (resource.name == resourceName) {
-                foundResource = resource.content;
-            }
-        });
+        return this;
     }
 
-    return foundResource;
-}
\ No newline at end of file
+    /**
+     * Returns the searched resource
+     *
+     * @param {string} resourceName The name of the resource
+     * @returns {Resource} The found resource
+     */
+    get(resourceName) {
+        if (!this._application || !this._application.resources) {
+            throw new Error(`Unable to fetch resource from "null" application`);
+        }
+
+        const foundResource = Java.from(this._application.resources).find(resource => resource.name === resourceName);
+
+        if (!foundResource) {
+            throw new Error(`Application "${this._application.name}" does not contain resource "resourceName"`);
```

Comment:
Shouldn't this print the variable `resourceName`?

Reply:
Yes forgot a `$`

Reply:
Done